faststats.js

```diff
@@ -181,12 +181,13 @@ Stats.prototype = {
 			else if(tuple[0] > 0 && (this.max === a || this.min === a)) {
 				var i = this.length-1;
 				if(i>=0) {
-					this.max = this.min = this.data[i--];
-					while(i-- >= 0) {
+					this.max = this.min = this.data[i];
```

Comment:
You're right about the double decrement, but I think you can leave the `i--` here for the following reason:

* we assign this.max & this.min to this.data[i]
* we do a post decrement so `i` is now `length-2`
* we now iterate through the remaining elements (length-2 -> 0) to find the actual min & max.
* we do not need to compare with `length-1` any more as the conditions are guaranteed to be false

The change in the decrement for the while loop is correct. You could also change it to a `for` loop as `for(; i>=0; i--)`